ceph.spec.in

```diff
@@ -295,6 +295,7 @@ Requires:      python%{_python_buildid}-requests
 Requires:      python%{_python_buildid}-setuptools
 Requires:      util-linux
 Requires:      xfsprogs
+Requires:      e2fsprogs
```

Comment:
The `Requires:` lines are in some semblance of alphabetical order. Could you move e2fsprogs up between "cryptsetup" and "findutils" please?